Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageLoader/QtImageLoader.cpp

```diff
@@ -57,6 +73,170 @@ namespace ImageProcessingAtom
+                            fclose(pf);
+                            return false;
+                        }
+
+                        if (tempChunkHeader >> 7) // repeat count
+                        {
+                            // just use the first 7 bits
+                            tempChunkHeader = (uint8_t(tempChunkHeader << 1) >> 1);
+
+                            if (fread(reinterpret_cast<char*>(&tempData), sizeof(char), ui32BpP / 8, pf) != ui32BpP / 8)
+                            {
+                                delete vui8Pixels;
+                                fclose(pf);
+                                return false;
+                            }
+
+                            for (int i = 0; i <= tempChunkHeader; i++)
+                            {
+                                vui8Pixels->at(tempByteIndex++) = tempData[0];
+                                vui8Pixels->at(tempByteIndex++) = tempData[1];
+                                vui8Pixels->at(tempByteIndex++) = tempData[2];
+                                if (ui32BpP == 32)
+                                {
+                                    vui8Pixels->at(tempByteIndex++) = tempData[3];
+                                }
+                                    
+                            }
+                        }
+                        else // data count
+                        {
+                            // just use the first 7 bits
+                            tempChunkHeader = (uint8_t(tempChunkHeader << 1) >> 1);
+
+                            for (int i = 0; i <= tempChunkHeader; i++)
+                            {
+                                if (fread(reinterpret_cast<char*>(&tempData), sizeof(char), ui32BpP / 8, pf) != ui32BpP / 8)
+                                {
+                                    delete vui8Pixels;
+                                    fclose(pf);
+                                    return false;
+                                }
+                                vui8Pixels->at(tempByteIndex++) = tempData[0];
+                                vui8Pixels->at(tempByteIndex++) = tempData[1];
+                                vui8Pixels->at(tempByteIndex++) = tempData[2];
+                                if (ui32BpP == 32)
+                                {
+                                    vui8Pixels->at(tempByteIndex++) = tempData[3];
+                                }
+                                    
+                            }
+                        }
+                    } while (tempByteIndex < ui32Size);
+                }
+                // not useable format
+                else
+                {
+                    delete vui8Pixels;
+                    fclose(pf);
+                    return false;
+                }
+                                
+                fclose(pf);
+
+                image = QImage(ui32Width, ui32Height, QImage::Format_RGB888);
```

Comment:
It seems it doesn't save the alpha channel data from the tga file. I think it should check the ui3dBpP to determinate if it need to create Format_RGB888 or Format_RGBA8888 to preserve alpha data